src/include/miopen/tensor.hpp

```diff
@@ -204,9 +204,13 @@ struct MIOPEN_INTERNALS_EXPORT TensorDescriptor : miopenTensorDescriptor
     unsigned GetNumDims() const;
 
     miopenDataType_t GetType() const;
+    // clang-format off
```

Comment:
I'll fix this later as there will be a lot of changes not related to this PR.

Reply:
With default formatting the code becomes unreadable. It is not possible to control attributes with clang-tidy-12 which we are using. We could update it to clang-tidy-15 on Ubuntu 22.04 which is currently the minimum supported version. This will allow us to control more options (excluding this one, unfortunately).

CC @junliume